src/md-to-html.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+throw() { 
+  echo "$@" 1>&2
+  exit 1
+}
+
+# Validate arguments
+if [ $# -eq 0 ]
+  then
+    throw "No arguments supplied. Please specify md files to convert as first argument and output directory as second argument"
+fi
+
+
+# ensure npx exists in path
+if ! [ -x "$(command -v npx)" ]; then
+  throw "npx is not available"
+fi
+
+# save args to variables
+inputFiles=$1
+destDir=$2
+
+# create dir, if destination directory does not exist
+if [ ! -d "$destDir" ]; then
+  mkdir -p "$destDir"
```

Comment:
Not a big deal, but with the `-p` flag, you don't need to check whether or not the dir exists.